<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Escorting</class>
 <widget class="QWidget" name="Escorting">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>511</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_acompanhamento_3">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Droid Sans'; font-size:16pt; font-weight:600;&quot;&gt;ACOMPANHAMENTO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget_acEvolucao_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_Busca_2">
      <attribute name="title">
       <string>Pacientes Admitidos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_108">
         <item>
          <widget class="QLabel" name="label_nomeBuscaP_4">
           <property name="text">
            <string>Nome</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_nomeP_2</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nomeP_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_prontP_7">
           <property name="text">
            <string>Prontuário</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_prontP_2</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_prontP_8">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_buscarP_5">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_67">
         <property name="text">
          <string>Informações do Paciente</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTableView" name="tv_PatienteDescrEvo"/>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_109">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Replicar Última Evolução</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_EVOLUT_ADD">
           <property name="text">
            <string>Evoluir Paciente</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="lw_ESCORTINGpatList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_diagnosticos_2">
      <attribute name="title">
       <string>Diagnósticos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableWidget" name="tw_DIAG2EVO">
          <column>
           <property name="text">
            <string>Prontuário</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Estado</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Diagnóstico</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Horário</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="cb_States">
             <item>
              <property name="text">
               <string>Mantido</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Melhorado</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Piorado</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Encerrado</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_horas_2">
             <property name="text">
              <string>Horário da Evolução</string>
             </property>
             <property name="buddy">
              <cstring>dateTimeEdit_horasEvDiag</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pb_EVOstates">
             <property name="text">
              <string>Evoluir Estado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dTime_EVO">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yyyy HH:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>353</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pb_CONT">
         <property name="text">
          <string>Limpar Campos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_observacoes_2">
      <attribute name="title">
       <string>Observações</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_118">
         <item>
          <widget class="QLabel" name="label_pacienteOb_2">
           <property name="text">
            <string>Paciente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pacienteObs_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_prontPaciente_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Prontuário</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_prontPaciente_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_ocorrencias_2">
         <property name="text">
          <string>Ocorrências</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_117">
         <item>
          <widget class="QPushButton" name="pushButton_concluirEvolucao_2">
           <property name="text">
            <string>Concluir Evolução</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit_campoObservacoes_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
